### PR TITLE
fix the context parsing logic in system config subcommand #5408

### LINKED ISSUE #5408: [mesheryctl] GETCONTEXTS API is returning single context, even though kubeconfig.yaml has multiple contexts.
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

See related PR - https://github.com/meshery/meshery/pull/5402:  GETCONTEXTS API is returning single context, even though kubeconfig.yaml has multiple contexts.

The [GetContexts() function](https://github.com/meshery/meshery/blob/d22158c7b21777aeb5f55119349af20f4b754465/mesheryctl/internal/cli/root/system/config.go#L38), which invokes Meshery Server's `/api/system/kubernetes/contexts` API, needs to consider multiple Kubernetes contexts. 

See `GetContextsFromK8SConfig()` at https://github.com/meshery/meshery/blob/9be2c0816e60f6d269caa5271f8b18cfd652f2f3/handlers/k8sconfig_handler.go#L138

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #5408 

**Notes for Reviewers**
For reference, here is the response from /api/system/kubernetes/contexts
this

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]
@Revolyssup do we have a MeshKit Error Code for a failure to unmarshall the kubeconfig?

File: mesheryctl/internal/cli/root/system/config.go [hunk 1]
```diff
@@ -61,24 +61,37 @@ func getContexts(configFile string) ([]string, error) {
...
+	if results == nil || results["contexts"] == nil {
 		errstr := "Error unmarshalling the context info, check " + configFile + " file"
 		return nil, errors.New(errstr)
 	}
-
-	log.Debugf("Unmarshalled get context API response: %s", results)
-	var contexts []string
-	for _, item := range results {
-		contexts = append(contexts, item["name"].(string))
+	contexts, ok := results["contexts"].([]interface{})
```
Comment (hexxdump) [inline mesheryctl/internal/cli/root/system/config.go:74]: Assuming that the API returns an array even if there is a single context.

**Review by hexxdump — APPROVED** [review]
LGTM

File: mesheryctl/internal/cli/root/system/config.go (hunk 1 above)
Comment (Revolyssup) [reply mesheryctl/internal/cli/root/system/config.go:74]: If there is a single context then the array will have one item

**Pull request merged**